examples/battleship/README.md

```diff
@@ -1,30 +1,146 @@
+
+## Functional / Implementation Details: 
+This section breaks down the functional aspects of Battleship Baseline and the value it adds. 
+
+## The Baseline Protocol
+Baselining is the practice of using zero knowledge cryptography techniques to prove that counterparties maintain the same information in their respective systems of record, and to prove attributes of that information to third parties, without revealing the actual information.
+
+The practice is essential for greater coordination between companies, because companies today must work together more closely than ever before, but they’ve never been so aware of the risks of leaking information between entities. The world needs a way to increase coordination, while also increasing compliance (by reducing information security risks otherwise posed by that level of coordination), to transform its balkanized, bespoke and brittle patchwork of information systems.
+
+Say hello the the Baseline Protocol - for more information, [see here](https://baseline-protocol.org )
 
-## Gameplay
-Players generate an organization and join a workgroup. Ships are privately placed on a coordinate grid on which each players ship location is marked. The position of the ship in relation to the coordinate grid is then used to generate current state. Each turn, players attempt to locate their opponents ship through a series of coordinate defined missile launches. Target coordinates and board state are used as inputs in a zero-knowledge proof to determine if the opponents ship is present in the target space. A hit will result in an updated board state and verified proof that is recorded as a leaf in a merkle tree via a transaction to a locally run blockchain. Misses simply progress gameplay. Once a players ship is sunk, the game ends and a final proof is recorded. Because each proof generation triggered by a hit contains board state and previous hits, the final game proof can be used to verify the validity of the entire game.
+Before baselining transforms the world of industry, developers need to get used to the approach, making it important to get started with something simple, like a game. And ideally a game that’s easy to implement, easy to create new features for (best way to learn), and easy to see how it mirrors the real world implementation. 
 
-![Battleship/Baseline Gameplay](/examples/battleship/battleship.drawio.svg)
+## Battleship: Baseline
+‘Battleship Baseline’ is an implementation of the classic search & destroy strategy guessing game, using the design pattern specified in the Baseline Protocol standard. This implementation is a great way to start baselining. It gives you a simple framework of where and how to baseline information between multiple parties, a complete set of code that works, and a lot of clues on how to extend the code to create new features and for other use cases. 
 
-## Shortcomings
+## Demo
+[See the demo here](https://www.youtube.com/watch?v=qyqqu2wNZe8)
```

Comment:
Can we move this section right after "How Baselining the battleship was achieved" section? I think it would be good to see the things in action right after the theory on baselining battleship.